frontend/02 Plain Vanilla/README.md

```diff
@@ -16,6 +19,606 @@ npm install
 
 ```
 
+- Exec tests in watch mode:
+
+```bash
+npm run test:watch
+
```

Comment:
Remove extra line.
(Same for L19, L52, L66, L90, L109, L130, L157, L184, L198, L222, L243, L263, L283, L317, L361, L391, L419, L452, L482, L511, L538, L565, L592, L619)